src/system/settings/default-settings.ts

```diff
@@ -0,0 +1,80 @@
+import type {
+  AudioSettings,
+  DisplaySettings,
+  GamepadSettings,
+  GeneralSettings,
+  UserFacingSettings,
+} from "#app/@types/Settings";
+import { BattleStyle } from "#app/enums/battle-style";
+import { CandyUpgradeDisplayMode } from "#app/enums/candy-upgrade-display";
+import { CandyUpgradeNotificationMode } from "#app/enums/candy-upgrade-notification-mode";
+import { DamageNumbersMode } from "#app/enums/damage-numbers-mode";
+import { EaseType } from "#app/enums/ease-type";
+import { EggSkipPreference } from "#app/enums/egg-skip-preference";
+import { ExpGainsSpeed } from "#app/enums/exp-gains-speed";
+import { ExpNotification } from "#app/enums/exp-notification";
+import { HpBarSpeed } from "#app/enums/hp-bar-speed";
+import { MoneyFormat } from "#app/enums/money-format";
+import { MusicPreference } from "#app/enums/music-preference";
+import { ShopCursorTarget } from "#app/enums/shop-cursor-target";
+import { UiTheme } from "#app/enums/ui-theme";
+import { PlayerGender } from "#enums/player-gender";
+import { UiWindowType } from "#enums/ui-window-type";
+
+export const defaultGeneralSettings: GeneralSettings = {
+  gameSpeed: 2,
+  hpBarSpeed: HpBarSpeed.DEFAULT,
+  expGainsSpeed: ExpGainsSpeed.DEFAULT,
+  partyExpNotificationMode: ExpNotification.DEFAULT,
+  skipSeenDialogues: false,
+  eggSkipPreference: EggSkipPreference.NEVER,
```

Comment:
```suggestion
  eggSkipPreference: EggSkipPreference.ASK,
```
Asking was the default [before](https://github.com/Despair-Games/poketernity/blob/0c56fc32b467de4aa12ba6ef982d1b351daa0cb7/src/system/settings/settings.ts#L320)